scripts/bucket_upload.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+import sys
+import argparse
+
+from cloudbridge.factory import CloudProviderFactory
+from cloudbridge.factory import ProviderList
+from cloudbridge.interfaces.exceptions import DuplicateResourceException
+
+def main(arguments):
+    parser = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+    parser.add_argument('filepath', help="File to upload", type=str)
+    parser.add_argument('destination', help="Destination in bucket", type=str)
+    parser.add_argument('-b', '--bucket', help="Name of destination bucket", type=str)
+    parser.add_argument('-p', '--provider',
+                        choices=[ProviderList.__dict__[p] for p in ProviderList.__dict__.keys() if all(x not in p for x in ['_', 'MOCK'])],
```

Comment:
Perhaps it's time we make ProviderList an enum? https://stackoverflow.com/questions/29503339/how-to-get-all-values-from-python-enum-class/29503414

I think this is just a legacy thing from python 2 days.